sklearn/feature_selection/sequential_feature_selector.py

```diff
@@ -0,0 +1,323 @@
+        - An object to be used as a cross-validation generator.
+        - An iterable yielding train, test splits.
+        For integer/None inputs, if the estimator is a classifier
+        and `y` is either binary or multiclass, `StratifiedKFold` is used.
+        In all other cases, `KFold` is used.
+
+    n_jobs : int (default=1)
+        The number of CPUs to use for cross validation.
+
+    pre_dispatch : int, or string (default: '2*n_jobs')
+        Controls the number of jobs that get dispatched
+        during parallel execution in cross_val_score.
+        Reducing this number can be useful to avoid an explosion of
+        memory consumption when more jobs get dispatched than CPUs can process.
+        This parameter can be:
+        - None, in which case all the jobs are immediately created and spawned.
+          Use this for lightweight and fast-running jobs,
+          to avoid delays due to on-demand spawning of the jobs
+        - An int, giving the exact number of total jobs that are spawned
+        - A string, giving an expression as a function
+            of `n_jobs`, as in `2*n_jobs`
+
+    Attributes
+    ----------
+    estimator_ : scikit-learn Classifier or Regressor
+        Fitted clone of `estimator`.
+
+    feature_subset_idx_ : array-like, shape = [n_predictions]
+        Feature Indices of the selected feature subsets.
+
+    score_ : float
+        Cross validation average score of the selected subset.
+
+    subsets_ : dict
+        A dictionary of selected feature subsets during the
+        sequential selection, where the dictionary keys are
+        the lengths k of these feature subsets. The dictionary
+        values are dictionaries themselves with the following
+        keys: 'feature_subset_idx' (tuple of indices of the feature subset)
+              'cv_scores' (list individual cross-validation scores)
+              'avg_score' (average cross-validation score)
+
+    Examples
+    --------
+    The following example shows how to use the sequential feature selector
+    with default settings to select a feature subset, consisting of
+    1 to 3 features, from iris. The selection criteria for this
+    feature subset is the average cross-validation performance
+    (cv=5 by default) of the `estimator` (here: KNN)
+    during the greedy forward selection search.
+
+        >>> from sklearn.feature_selection import SequentialFeatureSelector
+        >>> from sklearn.neighbors import KNeighborsClassifier
+        >>> from sklearn.datasets import load_iris
+        >>> iris = load_iris()
+        >>> X, y = iris.data, iris.target
+        >>> knn = KNeighborsClassifier(n_neighbors=3)
+        >>> sfs = SequentialFeatureSelector(knn, n_features_to_select=(1, 3))
+        >>> sfs = sfs.fit(X, y)
+        >>> sfs.score_  # doctest: +ELLIPSIS
+        0.9733...
+        >>> sfs.feature_subset_idx_
+        (0, 2, 3)
+        >>> sfs.transform(X).shape
+        (150, 3)
+
```

Comment:
See also RFE, rfecv, SelectFromModel and reciprocated

Reply:
sry, what would your suggested change be in this case?

Reply:
All I mean is to Add a See Also section

Reply:
Oh, I see! Added that one plus a reference section.